frontend/src/components/details/CanvasComponent.jsx

```diff
@@ -8,35 +8,44 @@ such as diagrams and tables
 import React, { Component } from 'react';
 import PieChart from './PieChartComponent';
 import ColumnChart from './ColumnChartComponent';
-import Table from './TableComponent'
-import './canvas.css';
-import {Container, Row, Col } from 'react-grid-system';
-
+import Table from './TableComponent';
+import { Container, Row, Col } from 'react-grid-system';
+import CompareCanvas from './CompareCanvas';
 
 class CanvasPage extends Component {
```

Comment:
```suggestion
class CanvasComponent extends Component {
```

It might be necessary to change this in some more places than this line here.